config/samples/flows_v1beta2_flowcollector.yaml

```diff
@@ -92,6 +92,11 @@ spec:
         cpu: 100m
       limits:
         memory: 800Mi
+    advanced:
+      secondaryNetworks:
+        - name: "my-vms/custom-nad"
+          # Any of: MAC, IP, Interface
+          index: [MAC]
```

Comment:
That should be commented by default I guess

Reply:
done